job/kill_task.sh

```diff
@@ -0,0 +1,5 @@
+source <%= obj.jobInfoPath %>
+
+if [ $IS_CONTAINER == true ]; then
+  exec_cmd "docker stop $JOB_ID"
+fi
```

Comment:
- add echo saying `killing container with id ...`
- add `else` saying `killing process with id ..`. this will act as a placeholder only for now

Reply:
also, dont we need `#!/bin/bash` at the top?

Reply:
We'll attach the headers before this for exec_cmd, so that will attach the #!/bin/bash.